submissions/RomanSaiko/oop/index.js

```diff
@@ -0,0 +1,72 @@
+
+class Inhabitant {
+    constructor ({ species, name, gender, saying, friends = "No friends! One man army.", paws, hands, legs }) {
+        this.species = species
+        this.name = name
+        this.gender = gender
+        this.saying = saying
+        this.friends = friends
+        this.paws = paws
+        this.hands = hands
+        this.legs = legs
+    }
+
+    toString() {
+        return `${this.species}; ${this.name}; ${this.gender}; ${this.saying}; ${this.friends}`;
```

Comment:
Nicely done.
However, there is pattern here.
Imagine you have dozens of props. It is easy to miss a space and/or a semicolon under current approach.
Also `this.whatever` looks a bit wordy.
How about listing property names, then mapping them against object values and finally joining them with a delimiter?